app/repos/people.js

```diff
@@ -493,6 +494,65 @@ const deletePerson = async (reference, user, transaction) => {
   await sendDeleteToAudit(PERSON, personWithRelationships, user)
 }
 
+const purgePersonByReferenceNumber = async (reference, user, transaction) => {
+  if (!transaction) {
+    return await sequelize.transaction(async (t) => purgePersonByReferenceNumber(reference, user, t))
+  }
+
+  const [person] = await sequelize.models.person.findAll({
+    order: [[sequelize.col('addresses.address.id'), 'DESC']],
+    where: { person_reference: reference },
+    paranoid: false,
+    include: [
+      {
+        model: sequelize.models.person_address,
+        as: 'addresses',
+        where: {
+          id: {
+            [Op.not]: null
```

Comment:
Not sure why you need this where clause

Reply:
I need to test this, not sure if it will work or not...but have noticed that sequelize sometimes only returns a single child, this was an attempt to try and force it to return multiple children

Reply:
I think there is a workaround elsewhere for this type of thing (only returning a single child where it should return multiples).